src/ui/components/index.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
-import { Container, BooleanInput, Label, SliderInput, VectorInput, SelectInput } from 'pcui';
+import {
+    BooleanInput,
+    ColorPicker as PCUIColorPicker,
```

Comment:
Weird that you have to do this. Can we rename the app's ColorPicker?

Reply:
yes!